README.rst

```diff
@@ -25,7 +25,7 @@ Setup
 
 Clone this repo to your workspace.  Before installing the client run::
 
-    $ mv src/lpconnector/config/config.ini.template src/lpconnector/config/config.ini
+    $ mv src/lpconnector/base/config/config.ini.template src/lpconnector/base/config/config.ini
```

Comment:
so this needs to moved to src/lpconnector/base/config/config.ini.  cool. this solves the problem then! Thanks